bundler/spec/commands/install_spec.rb

```diff
@@ -545,6 +545,8 @@
 
          DEPENDENCIES
 
+         CHECKSUMS
```

Comment:
do we want to print this section when it's empty?

Reply:
🤷‍♀️ I'm not sure. Maybe it's helpful to have an obvious way to see that we were unable to get checksums? I could go either way

Reply:
Had the same thought, you ended up deciding empty section was better?